azure/packages/test/end-to-end-tests/package.json

```diff
@@ -37,13 +37,13 @@
 		"all": true,
```

Comment:
Not for this PR, but could we remove most of this boilerplate from our many packages by extending some base `c8` config file?

Reply:
Quick check suggests [it might be possible](https://github.com/istanbuljs/nyc?tab=readme-ov-file#adding-your-overrides).